packages/components/src/SpectrumProvider.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Provider, ProviderProps } from '@adobe/react-spectrum';
+import Log from '@deephaven/log';
+import { themeDHDefault } from './SpectrumUtils';
+
+const log = Log.module('SpectrumProvider');
+
+export interface SpectrumProviderProps {
+  colorScheme?: 'dark' | 'light';
+  theme?: ProviderProps['theme'];
+  /**
+   * Since this provider may be used to wrap components that are not at the root
+   * of the application, we may want to leave the background color alone so that
+   * components can "inherit" parent component background colors.
+   */
+  transparentBackground?: boolean;
```

Comment:
Any reason to only include a `transparentBackground` flag and not a `UNSAFE_style` prop to match the `Provider` ?

Reply:
Looking at this more closely, I'm wondering if the custom provider is providing any real value. Maybe it makes more sense to just do

```typescript
<Provider
  UNSAFE_style={{ backgroundColor: 'transparent' }}
  colorScheme="dark"
  theme={themeDHDefault}
>
```

Reply:
I think we only need the `themeDHDefault` and don't need the custom provider at all. This will be used for wrapping the ACL panels in Enterprise. My assumption is that we will soon be creating more Spectrum based components in @deephaven/components, so having the themeDHDefault be defined there makes the most sense to me.

Reply:
@mofojed  I went ahead and removed the custom `SpectrumProvider` (I kept the `SpectrumUtils`)